File: JavaScript/프로그래머스/0/181892. n 번째 원소부터/n 번째 원소부터.js
function solution(num_list, n) {
    var answer = new Array(num_list.length-n+1);
    var m=0;
    for(var i=n-1; i<num_list.length ;i++){
        answer[m] = num_list[i];
        m++;
    }    
    return answer;
}

File: 프로그래머스/0/181892. n 번째 원소부터/n 번째 원소부터.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list, int n) {
    vector<int> answer(size(num_list)-n+1);
    int m=0;
    for(int i=n-1; i<size(num_list); i++){
        answer[m] = num_list[i];
        m++;
    } 
    return answer;
}